platforms/Windows/bundle/theme.wxl

```diff
@@ -0,0 +1,81 @@
+
+<WixLocalization Culture="en-us" Language="1033" xmlns="http://wixtoolset.org/schemas/v4/wxl">
+  <String Id="Caption" Value="[WixBundleName] Setup" />
+  <String Id="Title" Value="[WixBundleName]" />
+  <String Id="CheckingForUpdatesLabel" Value="Checking for updates" />
+  <String Id="UpdateButton" Value="&amp;Update to version [WixStdBAUpdateAvailable]" />
+  <String Id="InstallHeader" Value="Welcome" />
+  <String Id="InstallMessage" Value="Setup will install [WixBundleName] on your computer. Click Install to continue or Cancel to exit." />
+  <String Id="InstallMessageOptions" Value="Setup will install [WixBundleName] on your computer. Click Install to continue, Options to set installation options, or Cancel to exit." />
+  <String Id="InstallVersion" Value="Version [WixBundleVersion]" />
+  <String Id="ConfirmCancelMessage" Value="Are you sure you want to cancel?" />
+  <String Id="ExecuteUpgradeRelatedBundleMessage" Value="Previous version" />
+  <String Id="HelpHeader" Value="Setup Help" />
+  <String Id="HelpText" Value="/install | /repair | /uninstall | /layout [directory]&#xA;Installs, repairs, uninstalls, or creates a complete local copy of the bundle in [directory]. Install is the default.&#xA;&#xA;/passive | /quiet&#xA;Displays minimal UI with no prompts or displays no UI and no prompts. By default UI and all prompts are displayed.&#xA;&#xA;/norestart&#xA;Suppress any attempts to restart. By default UI will prompt before restart.&#xA;&#xA;/log [path\to\logfile.log]&#xA;Logs to a specific file. By default a log file is created in %TEMP%." />
+  <String Id="HelpCloseButton" Value="&amp;Close" />
+  <String Id="InstallLicenseLinkText" Value="[WixBundleName] &lt;a href=&quot;#&quot;&gt;license terms&lt;/a&gt;." />
+  <String Id="InstallAcceptCheckbox" Value="I &amp;agree to the license terms and conditions" />
+  <String Id="InstallOptionsButton" Value="&amp;Options" />
+  <String Id="InstallInstallButton" Value="&amp;Install" />
+  <String Id="InstallCancelButton" Value="&amp;Cancel" />
+  <String Id="OptionsHeader" Value="Setup Options" />
+  <String Id="OptionsLocationLabel" Value="Install location" />
+  <String Id="OptionsBrowseButton" Value="&amp;Browse" />
+
+  <String Id="OptionsFeaturesLabel" Value="Choose the features you want to install" />
+  <String Id="OptionsInstallBld" Value="Swift Build Tools" />
+  <String Id="OptionsInstallCli" Value="Swift Command Line Tools" />
+  <String Id="OptionsInstallDbg" Value="Swift Debugging Tools" />
+  <String Id="OptionsInstallIde" Value="Swift IDE Integration Tools" />
+  <String Id="OptionsInstallRtl" Value="Swift Utilities Windows" />
+  <String Id="OptionsInstallSdk" Value="Swift Windows SDK" />
+  <String Id="OptionsInstallRedistX86" Value="Swift Windows Redistributables (x86)" />
+  <String Id="OptionsInstallRedistAMD64" Value="Swift Windows Redistributables (AMD64)" />
+  <String Id="OptionsInstallRedistArm64" Value="Swift Windows Redistributables (Arm64)" />
```

Comment:
Can these descriptions be pulled in from the shared.wxl file?

Reply:
Unfortunately, no. The BA only supports one .wxl file so everything the BA needs to be in one. We could use this .wxl file for the shared MSI strings; it's a little dirty because this .wxl has so many extra strings than needed for the MSIs, but...Let me know if you want me to make that change.

Reply:
Well, I'm slightly concerned about changes like https://github.com/apple/swift-installer-scripts/pull/227 which would possibly miss some of the duplicated strings.

Reply:
Sure, I'll merge them.